app/services/get-capacity-view.js

```diff
@@ -35,11 +35,17 @@ module.exports = function (id, capacityDateRange, organisationLevel) {
 
 var parseCapacityBreakdown = function (workloadReports, organisationLevel) {
   var capacityBreakdown = []
+  var totals = { name: 'Total/Average', capacity: 0, totalCases: 0, totalARMS: 0, totalGs: 0, totalCMS: 0, totalSDRs: 0, totalParoms: 0, totalSdrConversions: 0, totalTotalT2aCases: 0 }
```

Comment:
Possibly update to `Total / Average` throughout with spaces?

Reply:
Yep, sure.